replica/database.cc

```diff
@@ -1119,6 +1119,17 @@ std::unordered_map<sstring, locator::vnode_effective_replication_map_ptr> databa
     return res;
 }
 
+std::set<sstring> database::get_tablets_keyspaces() const {
```

Comment:
probably, `std::vector<sstring>` is good enough for this purpose. as the names are unique in `_keyspaces` and we don't need to find a name by its key after filtering the names of keyspaces.

Reply:
right. makes sense